utils/build-script-impl

```diff
@@ -1799,6 +1799,10 @@ for deployment_target in "${HOST_TARGET}" "${CROSS_TOOLS_DEPLOYMENT_TARGETS[@]}"
                 LLVM_BIN="$(build_directory_bin ${deployment_target} llvm)"
                 NINJA_BIN="ninja"
 
+                if [[ "${USE_GOLD_LINKER}" ]]; then
+                    USE_GOLD_LINKER="-fuse-ld=gold"
```

Comment:
Changing the value of this variable from the expected 0 or 1 looks like a hack that might lead to surprises later.